v2/internal/typescriptify/typescriptify.go

```diff
@@ -920,7 +955,7 @@ func indentLines(str string, i int) string {
 
 func getStructFQN(in string) string {
 	result := strings.ReplaceAll(in, "[]", "")
-	result = strings.ReplaceAll(result, "*", "")
+	//result = strings.ReplaceAll(result, "*", "")
```

Comment:
This is a question mark for me. I believe with the pointer unwrapping it is unnecessary now but it needs to match the same name generation as in binding.go